specifications/api/swagger.yaml

```diff
@@ -1851,6 +1959,85 @@ components:
         state: *walletState
         tip: *blockReference
 
+    ApiActiveSharedWallet: &ApiActiveSharedWallet
+      type: object
+      required:
+        - id
+        - name
+        - account_public_key
+        - account_ix
+        - address_pool_gap
+        - payment_script_template
+        - balance
+        - delegation
+        - state
+        - tip
+      properties:
+        id: *walletId
+        name: *walletName
+        account_public_key: *walletAccountPubkey
+        account_ix: *derivationSegment
+        address_pool_gap: *walletAddressPoolGap
+        payment_script_template: *scriptTemplate
+        delegation_script_template: *scriptTemplate
+        balance: *walletBalance
+        delegation: *ApiWalletDelegation
+        state: *walletState
+        tip: *blockReference
```

Comment:
* `passphrase` is missing from here, which will be necessary for wallet restored from a mnemonic. 
* `assets` is missing, shared wallets are multi-assets wallets too
* I don't think `account_public_key` should be there, we already have endpoints for retrieving keys of wallets so let's not duplicate information.

Reply:
indeed. done in https://github.com/input-output-hk/cardano-wallet/pull/2549/commits/5d9014e17e541e518db0783bad52b021a8c8b504